apps/docs/src/app/platform/component-docs/platform-forms/platform-time-picker/examples/platform-time-picker-template-example.component.ts

```diff
@@ -0,0 +1,25 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+import { TimeObject } from '@fundamental-ngx/core';
+
+@Component({
+    selector: 'fdp-time-picker-template-example',
+    templateUrl: './platform-time-picker-template-example.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PlatformTimePickerTemplateExampleComponent {
+    time24h: TimeObject = { hour: 18, minute: 0, second: 0 };
```

Comment:
Since new `DatetimeAdapter` has been introduced there is no `TimeObject` anymore instead we should use FdDatetime instace (it's common for time and date components)